musescore/MuseScore
# 6-key Braille input

Implement the ability to enter notes in the Braille panel and have them rendered in the score.

Demo: https://youtu.be/xnD5Py4YbfI?t=206

To test:

1. Enable the Braille panel in **Preferences > Braille**.
2. Create a blank score.
3. Navigate to a rest in an empty measure.
4. Tab to the Braille panel.
5. Press `N` to begin note input.
6. Press and release the `S`, `F` and `K` keys all at the same time.

Result:
- In the Braille panel, the ⠕ character (Braille dots 1, 3, 5) appears.
- In the score, a half-note D appears.
